RecoLocalTracker/SiPixelRecHits/plugins/alpaka/PixelCPEFastParamsESProducerAlpaka.cc

```diff
@@ -42,6 +42,7 @@ namespace ALPAKA_ACCELERATOR_NAMESPACE {
 
     edm::ParameterSet pset_;
     bool useErrorsFromTemplates_;
+    bool GoodEdgeAlgo_;
```

Comment:
Naming rule nr 9 in https://cms-sw.github.io/cms_coding_rules.html
```suggestion
    bool goodEdgeAlgo_;
```